src/java/main/org/apache/zookeeper/server/quorum/FastLeaderElection.java

```diff
@@ -86,6 +86,7 @@
 
     QuorumCnxManager manager;
 
+    private SyncedLearnerTracker leadingVoteSet;
```

Comment:
it's not totally clear what this is used for. can you comment it?

Reply:
It is used to pass into the Leader after the Leader instance is constructed.